static/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -259,7 +259,9 @@ class AddDashboardWidgetModal extends React.Component<Props, State> {
   handleAddSearchConditions = () => {
     this.setState(prevState => {
       const newState = cloneDeep(prevState);
-      newState.queries.push(cloneDeep(newQuery));
+      const query = cloneDeep(newQuery);
+      query.fields = this.state.queries[0].fields;
```

Comment:
I don't think we'd ever have a state where there isn't a query?

Reply:
Yeah it's not possible to delete the first query from the widget so this looks safe